lib/redux/actions/async.js

```diff
@@ -538,13 +582,37 @@ export function retrieveTargetsFromStorage() {
   };
 }
 
+export function createCustodialAccount(profile) {
+  return (dispatch, getState) => {
+    const { api } = services;
+    dispatch(syncActions.createCustodialAccountRequest());
+    api.user.createCustodialAccount(profile, (err, account) => {
+      if (err) {
+        dispatch(syncActions.createCustodialAccountFailure(err));
+      }
+      else {
+        dispatch(syncActions.createCustodialAccountSuccess(account));
```

Comment:
And it does seem, upon reflection that the full refresh is probably just hitting the problem with a large hammer. I'll see about just updating `allUsers` appropriately.